folium/folium.py

```diff
@@ -58,7 +63,7 @@ class Map(object):
     def __init__(self, location=None, width=960, height=500,
                  tiles='OpenStreetMap', API_key=None, max_zoom=18, min_zoom=1,
                  zoom_start=10, attr=None, min_lat=-90, max_lat=90,
-                 min_lon=-180, max_lon=180):
+                 min_lon=-180, max_lon=180,playback=False):
```

Comment:
I am starting to get a little bit worried about the kwargs in the `__init__`.  We'll end up like `pandas.read_wathever` tons of options :smile:

Maybe we can brainstorm a better way to start maps, something that we could pass the advanced options/objects and added them to the simple map.